modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -341,12 +353,14 @@ FeatureFloodCount::buildLocalToGlobalIndices(std::vector<unsigned int> & local_t
   mooseAssert(_is_master, "This method must only be called on the root processor");
 
   counts.resize(_n_procs, 0);
```

Comment:
resize only initializes the new entries with 0. It is not obvious from the code below if each vector element gets written to. For clarity I'd suggest either removing th `, 0` or changing the `resize` to an `assign`, depending on whether you need the zeroing.